spotlight-client/src/DataStore/utils.ts

```diff
@@ -30,6 +30,7 @@ export function getTenantFromDomain(): TenantId | undefined {
   // production domains
   if (domain.endsWith(".gov")) {
     // TODO(#530): Add ID here
+    // TODO(#): Add ME here
```

Comment:
Let's create an actual issue to track this?

Reply:
Created an issue that mirrors #540: https://github.com/Recidiviz/public-dashboard/issues/569 and updated the code to point to it.

Looks like this is what code we need to update to point to the correct production domain?